client/src/components/Modals/SubmitPitch/index.tsx

```diff
@@ -66,53 +67,60 @@ const SubmitPitchModal: FC<SubmitPitchModalProps> = ({
   const [title, setTitle] = useState('');
   const [description, setDescription] = useState('');
   const [link, setLink] = useState('');
-  const [topics, setTopics] = useState(new Set<string>());
+  const [topics, setTopics] = useState<string[]>([]);
   const [conflictofInterest, setConflictOfInterest] = useState<
     boolean | undefined
   >();
   const [didSubmit, setDidSubmit] = useState(false);
+  const [writerChoice, setWriterChoice] = useState<boolean | undefined>();
```

Comment:
There's probably a reason you gave `writerChoice` the type of `boolean | undefined`, but I'm just wondering if you could change it to just the type `boolean` and make the default value `false`, as that would simplify some logic in the HTML.

Reply:
That would automatically select a form value in the field which I'm not sure if we want to do, However, you Andrew is right that you don't need the undefined type in the useState declaration because it will automatically gain this type if you don't assign a default value. Will review the rest of this PR later.